package.json

```diff
@@ -29,27 +29,27 @@
   },
   "dependencies": {
     "@hapi/boom": "9.x.x",
-    "@hapi/call": "7.x.x",
+    "@hapi/call": "8.x.x",
     "@hapi/hoek": "9.x.x",
-    "@hapi/joi": "17.x.x",
-    "lodash": "^4.17.15"
+    "joi": "17.x.x",
+    "lodash": "^4.17.19"
   },
   "peerDependencies": {
-    "@hapi/hapi": ">=19",
+    "@hapi/hapi": ">=20",
```

Comment:
If you want to keep this as a patch version you may want to keep the peer dependency loose to indicate support for both v19 and v20.  Otherwise npm will bug you when using it with hapi v19.